### PR TITLE
Move status code errors to `httpscerr` module

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/4535
## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests
- [ ] Updated the spec
- [x] Checked native-image compatibility

### CODE REVIEW

**Review by TharmiganK — APPROVED** [review]
LGTM

File: ballerina/modules/http_status/http_status_errors.bal [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```
Comment (TharmiganK) [inline ballerina/modules/http_status/http_status_errors.bal:1]: Why this is changed?

File: ballerina/modules/http_status/init.bal [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+function init() {
```
Comment (TharmiganK) [inline ballerina/modules/http_status/init.bal:19]: Do we need this file? This calls the same native function

Comment (dilanSachi) [reply ballerina/modules/http_status/init.bal:19]: We need it to init the http status module

Comment (dilanSachi) [reply ballerina/modules/http_status/init.bal:19]: it calls `setHttpStatusModule` native function

File: ballerina/modules/http_status/http_status_errors.bal (hunk 1 above)
Comment (dilanSachi) [reply ballerina/modules/http_status/http_status_errors.bal:1]: AFAIR in latest license headers we add this as `.com` right?

**Pull request merged**